plugins/inputs/ravendb/ravendb_test.go

```diff
@@ -0,0 +1,435 @@
+		"cpu_machine_usage":                                             41.0779,
+		"cpu_machine_io_wait":                                           2.55,
+		"cpu_processor_count":                                           8,
+		"cpu_assigned_processor_count":                                  7,
+		"cpu_thread_pool_available_worker_threads":                      32766,
+		"cpu_thread_pool_available_completion_port_threads":             1000,
+		"memory_allocated_in_mb":                                        235,
+		"memory_installed_in_mb":                                        16384,
+		"memory_physical_in_mb":                                         16250,
+		"memory_low_memory_severity":                                    0,
+		"memory_total_swap_size_in_mb":                                  1024,
+		"memory_total_swap_usage_in_mb":                                 456,
+		"memory_working_set_swap_usage_in_mb":                           89,
+		"memory_total_dirty_in_mb":                                      1,
+		"disk_system_store_used_data_file_size_in_mb":                   28,
+		"disk_system_store_total_data_file_size_in_mb":                  32,
+		"disk_total_free_space_in_mb":                                   52078,
+		"disk_remaining_storage_space_percentage":                       22,
+		"license_type":                                                  "Enterprise",
+		"license_expiration_left_in_sec":                                25466947.94,
+		"license_utilized_cpu_cores":                                    8,
+		"license_max_cores":                                             256,
+		"network_tcp_active_connections":                                84,
+		"network_concurrent_requests_count":                             1,
+		"network_total_requests":                                        3,
+		"network_requests_per_sec":                                      0.03322,
+		"network_last_request_time_in_sec":                              0.0265,
+		"network_last_authorized_non_cluster_admin_request_time_in_sec": 0.04,
+		"certificate_server_certificate_expiration_left_in_sec":         104,
+		"certificate_well_known_admin_certificates":                     "a909502dd82ae41433e6f83886b00d4277a32a7b;4444444444444444444444444444444444444444",
+		"cluster_node_state":                                            4,
+		"cluster_current_term":                                          28,
+		"cluster_index":                                                 104,
+		"databases_total_count":                                         25,
+		"databases_loaded_count":                                        2,
+	}
+
+	serverTags := map[string]string{
+		"url":               ts.URL,
+		"node_tag":          "A",
+		"cluster_id":        "6b535a18-558f-4e53-a479-a514efc16aab",
+		"public_server_url": "http://raven1:8080",
+	}
+
+	compareFields(t, serverFields, acc, "ravendb_server")
+	compareTags(t, serverTags, acc, "ravendb_server")
```

Comment:
No need to do this. Check `testutil.RequireMetricEqual()` and `testutil.RequireMetricsEqual()`! You can also ignore the time by setting the `IgnoreTime` option.

Reply:
done - thanks for this method

Reply:
Thx, please mark as resolved.